### PR TITLE
Feature/save component position

### PR DESCRIPTION
**Pull request by sohzaz**
:information_source:  Don't forget to modify the changelog.md before merging this branch.

### CODE REVIEW

File: src/models/ComponentDrawOption.js [hunk 1]
```diff
@@ -43,6 +47,7 @@ class ComponentDrawOption {
      * @type {Number}
      */
     this.height = height || null;
+    this.needsResizing = needsResizing || false;
```
Comment (Zorin95670) [inline src/models/ComponentDrawOption.js:50]: missing test

File: tests/unit/draw/DefaultDrawer.spec.js [hunk 2]
```diff
@@ -545,4 +602,39 @@ describe('Test Class: DefaultDrawer()', () => {
...
+      drawer = new DefaultDrawer();
+    });
+
+    it('Should flag the starting node', () => {
+      const startNode = { data: { drawOption: new ComponentDrawOption() } };
+
+      drawer.__markAsNeedingResize(startNode);
+      expect(startNode.data.drawOption.needsResizing).toEqual(true);
+    });
+    it('Should flag the parent nodes', () => {
```
Comment (Zorin95670) [inline tests/unit/draw/DefaultDrawer.spec.js:619]: missing space before it

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: src/models/ComponentDrawOption.js (hunk 1 above)
Comment (sohzaz) [reply src/models/ComponentDrawOption.js:50]: Added

File: tests/unit/draw/DefaultDrawer.spec.js (hunk 2 above)
Comment (sohzaz) [reply tests/unit/draw/DefaultDrawer.spec.js:620]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**